command/server/server.go

```diff
@@ -242,6 +242,13 @@ func setFlags(cmd *cobra.Command) {
 		"maximum size in bytes for a message read from the peer by websocket",
 	)
 
+	cmd.Flags().DurationVar(
+		&params.rawConfig.RelayerTrackerPollInterval,
```

Comment:
I think it is enough to validate it only for relayer node. If it is not a relayer node, it doesn't matter what the user specifies, because it won't be used.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1887/commits/f47c9ae2d3128f5fa8fa5bd00b49ef665ae38f18